src/interfaces/IPoolManager.sol

```diff
@@ -151,8 +151,13 @@ interface IPoolManager is IFees, IERC1155 {
         int256 liquidityDelta;
     }
 
-    /// @notice Modify the position for the given pool
-    function modifyPosition(PoolKey memory key, ModifyPositionParams memory params, bytes calldata hookData)
+    /// @notice Mint a new position in a pool
+    /// @dev Poke by calling with a zero liquidityDelta
+    /// @param key The pool to mint a position in
+    /// @param params The parameters of pass into modifyPosition
+    /// @param hookData Any data to pass to the callback, via `ILockCallback(msg.sender).lockAcquired(data)`
+    /// @return delta The balance delta of the position minted
+    function mintPosition(PoolKey memory key, ModifyPositionParams memory params, bytes calldata hookData)
```

Comment:
consider taking in different params than the underlying modifyPosition, i.e using uint256 for mint and burnPosition but casting to int for burn